ui/src/main/webapp/src/app/misc/about.component.ts

```diff
@@ -0,0 +1,30 @@
+import {Component, OnInit, AfterViewInit} from "@angular/core";
+
+@Component({
+    templateUrl: './about.component.html',
+    styles: [`
+        /deep/ img.windupContributor {
+            width: 35px;
+            margin: 4px;
+        }
+    `]
+})
+export class AboutPageComponent implements OnInit, AfterViewInit {
+
+    ngOnInit():any {
+    }
+
+    ngAfterViewInit(): void {
+        var divTarget = $("#windup-contributors");
```

Comment:
Haha, yeah, right now, non user impacting"technical debt" type things are the least of our worries.